tests/network/sriov.go

```diff
@@ -671,11 +671,11 @@ func findIfaceByMAC(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineIns
 	return ifaceName, nil
 }
 
-func newSRIOVVmi(networks []string, cloudInitNetworkData string) *v1.VirtualMachineInstance {
-	options := []libvmi.Option{
+func newSRIOVVmi(networks []string, cloudInitNetworkData string, opts ...libvmi.Option) *v1.VirtualMachineInstance {
```

Comment:
This does not look good now: Some options are added in this function, some are passed from outside.
But for the ones passed from outside, it is just for a single option.

While I cannot ask to inline this one, as it is used in many places, you could do this small trick:

```go
vmi := newSRIOVVmi([]string{networkName}, cloudInitNetworkDataWithStaticIPsByMac(networkName, mac.String(), cidr))
addNodeAffinity = libvmi.WithNodeAffinityFor(nodeName)
addNodeAffinity(vmi)
```

It is a bit hacky, but IMO much better than the current suggestion.